addons/medical_blood/functions/fnc_spurt.sqf

```diff
@@ -0,0 +1,47 @@
+/*
+ * Author: Sickboy
+ * Spurt blood on the ground
+ *
+ * Arguments:
+ * 0: unit <OBJECT>
+ * 1: direction <NUMBER>
+ * 2: damage <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [UNIT, random 360, 1] call ace_medical_blood_fnc_spawnBlood
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+#define MAXIMUM_DROPS 4
+#define DISTANCE_BETWEEN_DROPS 0.20
+#define OFFSET 0.25
+
+params ["_unit", "_dir", "_damage"];
+
+private _pos = getPosATL _unit;
+private _distanceBetweenDrops = DISTANCE_BETWEEN_DROPS * _damage;
+private _offSet = OFFSET + _distanceBetweenDrops;
+private _sinDir = sin _dir;
+private _cosDir = cos _dir;
+
+_pos params ["_x", "_y"];
+
+private _bloodPos = [_x + (_sinDir * _offSet), _y + (_cosDir * _offSet), 0];
```

Comment:
should use the alternative syntax of `getPos`

`origin getPos [distance, heading]`

Note: that `origin getPos` reports `AGL` so convert to `ASL`